test/expected-draft/paper.tex

```diff
@@ -136,9 +136,9 @@
 \renewcommand{\footrulewidth}{0.25pt}
 
 \fancyfoot[L]{\parbox[t]{0.98\headwidth}{\footnotesize{\sffamily Krewinkel
-et al. (1970). Article Writing with Markdown and the Open Journals
+et al. (2024). Article Writing with Markdown and the Open Journals
```

Comment:
the 2024 here is not ideal, but I am running out of steam on figuring out how to make this work in general